### DISCUSSION TITLE
Is Granite 3.0 suppots Japanese Language

### DISCUSSION THREAD

**Discussion by Kazuki-masuda** [ibm-granite/discussions#15] [Q&A] [answered]
Is Granite 3.0 suppots Japanese Language?

**Comment by LKamogawa** [comment, marked as answer]
Hi, Masuda-San!

Yes! Granite 3.0 supports Japanese language.
In the [Download Model Page](https://huggingface.co/ibm-granite/granite-3.0-8b-instruct) of the Official Granite Hugging Face, Japanese is listed as one of the supported languages.

Also, I asked your question to Granite 3.0, and it answered **“Yes!”**
![image](https://github.com/user-attachments/assets/cc792988-bc3e-49b5-bc25-55c28c3e0cde)

**Reply by Kazuki-masuda** [reply]
Thank you Kamogawa san, I will try it.
